packages/terra-date-input/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Fixed
```

Comment:
This was added to the wrong packages changelog, I believe this was ment for terra-date-time-picker, that's why the danger script is failing.

Reply:
Oh thanks!